### DISCUSSION TITLE
Can I bootstrap one table instead of re-boostrapping entire db?

### DISCUSSION THREAD

**Discussion by qbzenker** [toluaina/pgsync#501] [Q&A]
Hi,

I've added a table to my schema. I'd like to bootstrap this one table instead of re-bootstrapping the entire db. Is this possible?

**Comment by mattdiez-at** [comment]
So far, I've not had great luck finding this answer either. 

Looks like you **may** be able to manually update the materialized view, the triggers, and the replication slot, but I'd recommend being very careful about attempting that.

**Reply by toluaina** [reply]
- Problem is we don't know in advance how this new table affects the existing relationships in your database.
- If it's a child table, for instance, your entire document structure in Elastic/Open Search needs to be updated.
- This is why we recommend reindexing if you change your schema.
- Optimizing the bootstrap and re-running is cheaper than resolving the impact of adding a new table to the schema.
